client/cmd/dexcctl/simnet-setup.sh

```diff
@@ -61,6 +64,11 @@ if [ $DOGE_ON -eq 0 ]; then
 	./dexcctl -p abc -p "" --simnet newwallet 3 dogecoindRPC ~/dextest/doge/alpha/alpha.conf
 fi
 
+if [ $FIRO_ON -eq 0 ]; then
+	echo configuring firo wallet
+	./dexcctl -p abc -p "" --simnet newwallet 3 firodRPC ~/dextest/firo/alpha/alpha.conf
```

Comment:
The "3" here is where the asset ID goes, so `136`

Reply:
Updated thanks, will push